docker-compose.yml

```diff
@@ -1,16 +1,20 @@
-version: '2'
+version: '3'
 
 services:
+
   web:
-    build: .
+    container_name: bookit_web
+    image: builditdigital/bookit-web:latest
     ports:
-     - "8080:80"
+     - "80:80"
+    links:
+      - "server:bookitapi"
     environment:
-    - API_BASE_URL=http://localhost:8888
+      - BOOKIT_API_BASE_URL=/api
```

Comment:
In the /dev version you have API_BASE_URL.  Are there really two different vars?.  I'm probably just missing something.